.github/workflows/R-CMD-check.yml

```diff
@@ -12,16 +12,19 @@ jobs:
       fail-fast: false
       matrix:
         config:
-        # - { os: windows-latest, r: 'release'}
-        # - { os: macOS-latest, r: 'release'}
-        # - { os: macOS-latest, r: 'devel'}
-        - { os: ubuntu-16.04, r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+        - { os: ubuntu-20.04, r: '4.0', args: "--no-manual" }
 
     env:
       R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
-      RSPM: ${{ matrix.config.rspm }}
 
     steps:
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2.3.0
```

Comment:
CI broken because the key is not available in secrets in this repo. Do you plan to add the ssh key and known hosts information here after merging?

Reply:
yes, I'll pass you the key and all the stuff, I asked DO people and we can run this project at zero cost (no abuse clause, of course) with that key, they liked the idea of extending tests to droplet creation, snapshotting ,etc.

Reply:
Sounds good